charts/agent/templates/_deployment-cluster-events-config.tpl

```diff
@@ -84,6 +85,23 @@ processors:
 
 {{- include "config.processors.attributes.k8sattributes.podcontroller" . | nindent 2 }}
 
+  # attributes to append to objects
+  attributes/observe_objects:
+    actions:
+      - key: observe_filter
```

Comment:
Why are we adding these keys?
They won't be preserved in Observe anyways.

Here is our direct-write entity table design: https://www.notion.so/observeinc/2024-07-k8s-direct-write-entity-table-schema-design-76a762812acf4e028e3f80ded51f0aa3?pvs=4#06d0261ed97740bfa9c1c57454db0e93
You can see we only look at `observe_transform`, we never look at `observe_filter`, let alone store it.

If you want to add debug information, let's just add it to `observe_transform.control`, and prefix it with `debug_`, like this: https://github.com/observeinc/helm-charts/blob/f3ec22a207f05c31fadd4eb885f4a6cea5ee9e81/charts/agent/templates/_deployment-cluster-events-config.tpl#L227